### PR TITLE
LPS-175317 Do not consider title field when mapping to a Standard object

### PR DESCRIPTION
**Pull request by guilhermedcamacho**

### CODE REVIEW

File: modules/apps/object/object-storage-salesforce/src/main/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/SalesforceObjectEntryManagerImpl.java [hunk 1]
```diff
@@ -482,7 +482,12 @@ private JSONObject _toJSONObject(
 				objectField.getExternalReferenceCode(),
 				Objects.equals(value, StringPool.BLANK) ? null : value);
 
-			if (Objects.equals(
+			String externalReferenceCode =
+				objectDefinition.getExternalReferenceCode();
+
+			if (Validator.isNotNull(externalReferenceCode) &&
+				externalReferenceCode.endsWith("__c") &&
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-storage-salesforce/src/main/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/SalesforceObjectEntryManagerImpl.java:489]: @guilhermedcamacho,

I believe we should define `"__c"` as a constant (`CUSTOM_OBJECT_SUFFIX`) in a class like SalesForceConstants or something like that.

I found that `__c` could be applied for custom field as well ([see](https://salesforce.stackexchange.com/questions/101844/what-are-the-object-and-field-name-suffixes-that-salesforce-uses-such-as-c-an)), but we can define one constant for now just for this use case.

What do you think?

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by guilhermedcamacho** [comment]
ci:test:sf

**Comment by guilhermedcamacho** [comment]
ci:test:relevant

**Comment by guilhermedcamacho** [comment]
ci:test:object

**Comment by guilhermedcamacho** [comment]
ci:reevaluate:1317401_7052

**Comment by MarinhoFeliphe** [comment]
Resent here: https://github.com/liferay-objects/liferay-portal/pull/1751

**Pull request closed without merging**